one/src/lib.rs

```diff
@@ -70,6 +70,16 @@ struct DaemonOpts {
     )]
     swarm_addresses: Vec<String>,
 
+    /// Address of bootstrap peers.
+    /// There are no default addresses, use this arg or the API to connect to bootstrap peers as needed.
+    #[arg(
+        long,
+        use_value_delimiter = true,
+        value_delimiter = ',',
+        env = "CERAMIC_ONE_BOOTSTRAP_ADDRESSES"
```

Comment:
From Nathaniel: Change to `CERAMIC_ONE_EXTRA_BOOTSTRAP_ADDRESSES` to concatenate additional bootstrap nodes to the official list. Makes it clear and also provides flexibility.